packages/e2eTesting/cypress/integration/FT/ActionableItem.js

```diff
@@ -0,0 +1,44 @@
+context('Actions', () => {
```

Comment:
Anyway we can have the cypress tests run on typescript?

Reply:
Apparently we do! [here](https://docs.cypress.io/guides/tooling/typescript-support#Install-TypeScript)